lib/checks/aria/has-widget-role.js

```diff
@@ -0,0 +1,6 @@
+var role = node.getAttribute('role');
```

Comment:
This check should have tests. Nothing too complicated, just check that it works.

Reply:
there are tests in this PR at test/checks/aria/has-widget-role.js

are there other things you're looking to test?

Reply:
Wow, I'm not sure how I missed that. My bad!